source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -451,45 +473,127 @@ void LinkParser::CaptureLinkToken()
     m_parsedResult.AppendToTokens(codeGen);
 }
 
+
+// list items have form of ^-\s+ or \r-\s+
+bool ListParser::MatchNewListItem(std::stringstream &stream)
+{
+    if (IsHyphen(stream.peek()))
+    {
+        stream.get();
+        if (stream.peek() == ' ')
+        {
+            stream.unget();
+            return true;
+        }
+        stream.unget();
+    }
+    return false;
+}
+
+// if lines are seperated by more than two new lines,
```

Comment:
>more than two new lines [](start = 29, length = 23)

It looks like this function will return true for a single newline...

Reply:
you are correct, before this method is get called, new line character was checked, so it make it two.

---
In reply to: [158146824](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158146824) [](ancestors = 158146824)